src/transaction/log_page_buffer.c

```diff
@@ -754,14 +754,16 @@ logpb_invalidate_pool (THREAD_ENTRY * thread_p)
 
   /*
    * Flush any append dirty buffers at this moment.
-   * Then, invalidate any buffer that it is not fixed and dirty
    */
   logpb_flush_pages_direct (thread_p);
 
+  /*
+   * Invalidate any buffer that is not fixed.
```

Comment:
I am not sure about the "that is not fixed" part of the comment.

Reply:
Yeah, the comment can be removed. There is no "fixing".